elasticapm/traces.py

```diff
@@ -451,6 +451,7 @@ class DroppedSpan(BaseSpan):
     def __init__(self, parent, leaf=False):
         self.parent = parent
         self.leaf = leaf
+        self.id = None
```

Comment:
`id` also needs to be added to `__slots__`

Reply:
Whoops, that was a silly mistake. Thanks for catching it!